frontend/src/components/taskList/taskListRow.tsx

```diff
@@ -276,6 +273,7 @@ export default function TableRowComponent(props: any) {
       AuthenticationService.getUsers().then((response) => {
         setUsers(response.data.data);
 
+        // eslint-disable-next-line array-callback-return
```

Comment:
"forEach() may be preferable when you’re not trying to change the data in your array, but instead want to just do something with it"
i wtedy pozbyć się // eslint-disable-next-line array-callback-return

Reply:
done